test/integration/datatypes_test.exs

```diff
@@ -61,8 +41,48 @@ defmodule DataTypesTest do
      uuid,
      varchar,
      varint)
-    VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)
+    VALUES
+    (#{Enum.map_join(1..16, ", ", fn _ -> "?" end)})
```

Comment:
This change came from #79, where it was unpleasant experience with adding more types iteratively.